src/project/DownloadFlashButton.tsx

```diff
@@ -35,21 +35,37 @@ interface DownloadFlashButtonProps {
  * flash (if WebUSB is supported) or otherwise just download a HEX.
  */
 const DownloadFlashButton = ({ size }: DownloadFlashButtonProps) => {
-  const connectionStatus = useConnectionStatus();
-  const connected = connectionStatus === ConnectionStatus.CONNECTED;
+  const status = useConnectionStatus();
+  const downloadOnly = status === ConnectionStatus.NOT_SUPPORTED;
+  const connected = status === ConnectionStatus.CONNECTED;
+  const variant = connected || downloadOnly ? "solid" : undefined;
+
   const actions = useProjectActions();
   const buttonWidth = "10rem"; // 8.1 with md buttons
   return (
     <HStack>
       <Menu>
         <ButtonGroup isAttached>
           {connected ? (
-            <FlashButton width={buttonWidth} mode={"button"} size={size} />
+            <FlashButton
+              variant={variant}
+              width={buttonWidth}
+              mode={"button"}
+              size={size}
+            />
           ) : (
-            <DownloadButton width={buttonWidth} mode={"button"} size={size} />
+            <DownloadButton
+              variant={variant}
+              width={buttonWidth}
+              mode={"button"}
+              size={size}
+              borderRight="1px"
```

Comment:
```suggestion
              borderRight={variant ? "0px" : "1px"}
```
This change keeps the gap consistent between the download/flash button and its additional actions consistent where WebUSB is not available. I.e., narrows the gap observed below to what it was previously.

![image](https://user-images.githubusercontent.com/95928279/159454435-28d02d11-d659-47f4-9a1e-07d7c5388b5c.png)

Reply:
Good catch, thank you.